CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.19.0] - 2021-03-11
+
+### Changed
+
+- Bump dusk-plonk to `0.6`
+- Bump dusk-hades to `0.14`
```

Comment:
As mentioned in the stand up, this would be better as:
```
- Change `dusk-plonk` from `0.5` to `0.6`
- Change `dusk-hades` from `0.13` to `0.14
```